build.rs

```diff
@@ -429,7 +429,7 @@ impl Target {
 
 fn build_c_code(target: &Target, out_dir: &Path) {
     let includes_modified = RING_INCLUDES.par_iter()
-        .weight_max()
+        .with_max_len(1)
```

Comment:
Could you give me a pointer to the documentation for this? It's not clear to me that these are the same thing or how they are different. Thanks!